### PR TITLE
Dump Pregnant Woman(pw) and Lactating Mother(lm) district wise for AP

### PR DESCRIPTION
**Pull request by mkangia**
two files were already present which were used to read ids and other related information and then consolidate that differently

### CODE REVIEW

**Comment by orangejenny** [comment]
killed build for this since it's closed

**Comment by mkangia** [comment]
yikes, sorry. Forgot the skip ci for this one, thanks @orangejenny

**Comment by mkangia** [comment]
fix this before running
```
person_cases_by_id.get(row[4], {}).get('husband_name', ''), # Husband's Name
person_cases_by_id.get(row[4], {}).get('rch_id'),
``` 
and add
```
row[4] not in person_cases_by_id # Missing in ES
```

File: custom/icds_reports/data_pull/ap_district_wise_pw_lm.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                    ['District', 'Project', 'Sector', 'AWC', 'Beneficiary Name', 'PW/LM', 'Husband Name',
+                     'Mother Phone Number', 'RCH Number'])
+                out_files[row[8]] = writer
+            writer.writerow([row[8], # District
+                            row[7], # Project
+                            row[6], # Sector
+                            row[5], # AWC
+                            row[2], # Beneficiary Name
+                            'PW', # PW/LM
+                            person_cases_by_id[row[4]].get('husband_name', ''), # Husband's Name
```
Comment (mkangia) [inline custom/icds_reports/data_pull/ap_district_wise_pw_lm.py]: fix this before running
```
person_cases_by_id.get(row[4], {}).get('husband_name', ''), # Husband's Name
person_cases_by_id.get(row[4], {}).get('rch_id'),
``` 
and add
```
row[4] not in person_cases_by_id # Missing in ES
```

**Pull request closed without merging**